edweiss-app/components/lectures/slides/StudentQuestion.tsx

```diff
@@ -0,0 +1,90 @@
+import TTouchableOpacity from '@/components/core/containers/TTouchableOpacity';
+import TView from '@/components/core/containers/TView';
+import Icon from '@/components/core/Icon';
+import TText from '@/components/core/TText';
+import FancyTextInput from '@/components/input/FancyTextInput';
+import { callFunction } from '@/config/firebase';
+import ReactComponent from '@/constants/Component';
+import LectureDisplay from '@/model/lectures/lectureDoc';
+import { t } from 'i18next';
+import { useState } from 'react';
+import { ActivityIndicator } from 'react-native';
+
+interface CustomDocument<T> {
+    data: T;
+}
+
+const StudentQuestion: ReactComponent<{ courseName: string, lectureId: string, questionsDoc: CustomDocument<LectureDisplay.Question>[] | undefined }> = ({ courseName, lectureId, questionsDoc }) => {
+    const [question, setQuestion] = useState<string>('');
+    const [isLoading, setIsLoading] = useState<boolean>(false);
+
+    // Display each question given as parameters as a component 
+    const renderQuestion = (question: string, key: React.Key) => (
+        <TView key={key} mb={'sm'} backgroundColor='crust' borderColor='surface0' radius={14} flex={1} flexDirection='column' ml='sm'>
+            <TText ml={16} mb={4} size={'sm'} pl={2} pt={'sm'} color='overlay2'>{t(`showtime:peer_question`)}</TText>
+            <TView pr={'sm'} pl={'md'} pb={'sm'} flexDirection='row' justifyContent='flex-start' alignItems='center'>
+                <TText ml={10} color='overlay0'>{question}</TText>
+            </TView>
+        </TView>
+    );
+
+    // Function for adding new question into the firebase storage
+    async function addQuestion(question: string) {
+        try {
+            await callFunction(LectureDisplay.Functions.createQuestion, {
+                courseId: courseName,
+                lectureId: lectureId,
+                question: question
+            });
+        } catch (error) {
+            console.error("Error creating question:", error);
+        }
+        setQuestion('');
+        setIsLoading(false);
+    }
```

Comment:
Nice, this is indeed very usefull and helpful, I did not consider it since it was still in the beginning and there were not many questions being displayed, but you are completely right that the sucess of sending a comment is not clear unless you go all the way searching for it. 

I will most certainly integrate this, I am planning to do my next PR correcting and completing some elements of the StudentQuestion component so this is in perfect timing